docker/Dockerfile-prow-env

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:bookworm
+
+LABEL maintainer="cloud-esf-dev@google.com"
```

Comment:
I'm not sure if we should use this email.

Reply:
what is this label for ?

Reply:
I think it just provides extra information for this image. We had this for espv2, so I copied it over but changed the email from "esp-eng@google.com" to "cloud-esf-dev@google.com".
I don't find anywhere that we are using this label, maybe @nareddyt can confirm.

Reply:
could you write a script or README.md on how to build this docker?

Reply:
Yes that's a good idea. Done!